contentcuration/contentcuration/viewsets/base.py

```diff
@@ -894,38 +896,98 @@ def create_change_tracker(pk, table, channel_id, user, task_name):
     # Clean up any previous tasks specific to this in case there were failures.
     meta = json.dumps(dict(pk=pk, table=table))
     TaskResult.objects.filter(channel_id=channel_id, task_name=task_name, meta=meta).delete()
+    progress = 0
+    status = states.STARTED
+
+    channel_layer = get_channel_layer()
+    room_group_name = channel_id
 
     task_id = uuid.uuid4().hex
-    task_object = TaskResult.objects.create(
-        task_id=task_id,
-        status=states.STARTED,
-        channel_id=channel_id,
-        task_name=task_name,
-        user=user,
-        meta=meta
+
+    async_to_sync(channel_layer.group_send)(
+        str(room_group_name),
+        {
+            'type': 'broadcast_tasks',
+            'tasks': {
+                'pk': pk,
+                'table': table,
+                'task_id': task_id,
+                'task_name': task_name,
+                'traceback': None,
+                'progress': progress,
+                'channel_id': channel_id,
+                'status': status,
+            }
+        }
     )
 
     def update_progress(progress=None):
         if progress:
-            task_object.progress = progress
-            task_object.save()
+            if progress == 100:
```

Comment:
What if this isn't called with `progress = 100`? It may be safer to broadcast this where the old task object was deleted, but I'm not sure we need to handle this as the old sync endpoint never returned `SUCCESS` tasks